ibm-mq-mixin/config.libsonnet

```diff
@@ -1,9 +1,13 @@
 {
   _config+:: {
+    enableMultiCluster: false,
+    ibmmqSelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
     dashboardTags: ['ibm-mq-mixin'],
     dashboardPeriod: 'now-1h',
     dashboardTimezone: 'default',
     dashboardRefresh: '1m',
+    logExpression: if self.enableMultiCluster then '{job=~"$job", cluster=~"$cluster"'
+    else '{job=~"$job"',
```

Comment:
I would rather have the `{` inside the dashboard code for consistency.

Reply:
Agreed, would make it a little simpler to maintain

Reply:
Agreed! Done! @Dasomeone @gaantunes